tests/console/docker_image.pm

```diff
@@ -34,7 +38,7 @@ sub run {
 
     install_docker_when_needed();
 
-    if (is_sle) {
+    if (is_sle && !isPublicCloud) {
```

Comment:
and why we need to skip it for PC ?

Reply:
The internal registries are not available on public-cloud, thus we skip registering them.